src/test/scala/nl/knaw/dans/easy/multideposit/actions/AddFileMetadataToDepositSpec.scala

```diff
@@ -0,0 +1,130 @@
+/**
```

Comment:
This was mostly fixed after a refactoring in `Action`. Looked at it by running `mvn test` and improved a couple of names. Should be better now. Can you check this again?